people/luan/index.html

```diff
@@ -12,17 +12,43 @@
             <div>
                 <h2 id="name">Luan's Demo Page</h2>
                 <span id="content">
-                Press cmd + shift + j to open developer tools
+                <h1></h1>
                 </span>
             </div>
             <br>
         </div>
 
+        <script src="//ajax.googleapis.com/ajax/libs/jquery/1.10.2/jquery.min.js"></script>
```

Comment:
This won't work from local if you use `//` because your protocol would be `file://`!
Try adding `http://ajax.googleapis.com/ajax/......`